man/systemd.network.xml

```diff
@@ -1139,6 +1139,15 @@
             <para>A boolean. Specifies whether the rule to be inverted. Defaults to false.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>Family=</varname></term>
+          <listitem>
+            <para>Takes a special value <literal>ipv4</literal> or <literal>ipv6</literal>.
```

Comment:
Yeah, it'd be nice to change this to: "Takes either "ipv4", "ipv6", or "both"." (and of course adjust the code to match that).